game-core/src/main/java/games/strategy/debug/error/reporting/UserErrorReport.java

```diff
@@ -1,35 +1,37 @@
 package games.strategy.debug.error.reporting;
 
-import java.util.logging.LogRecord;
+import java.util.Optional;
+
+import javax.annotation.Nullable;
 
 import org.triplea.http.client.error.report.create.ErrorReport;
-import org.triplea.http.client.error.report.create.ErrorReportDetails;
 
 import games.strategy.engine.ClientContext;
-import lombok.AccessLevel;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
 
 /**
- * Represents the data a user has entered into a swing GUI.
+ * The user error report window can be seeded with information from the console or an error that
+ * impacted the user. The user is given a chance to enter description information of their own. This
+ * data object represents the bundle of that data after a user has entered data.
+ * .
+ * Or, a user can open a new bug report and simply enter a description and submit it.
  */
 @Builder
-@EqualsAndHashCode
-@ToString
-@Getter(AccessLevel.PACKAGE)
 class UserErrorReport {
-  private final String title;
+
+  @Nullable
   private final String description;
-  private final LogRecord logRecord;
+  @Nullable
+  private final String errorData;
 
   ErrorReport toErrorReport() {
-    return new ErrorReport(ErrorReportDetails.builder()
-        .gameVersion(ClientContext.engineVersion().getExactVersion())
-        .logRecord(logRecord)
-        .title(title)
-        .description(description)
-        .build());
+    return ErrorReport.builder()
+        .operatingSystem(System.getProperty("os.name"))
```

Comment:
There are helpers for this kind of stuff in `g.s.engine.framework.system.SystemProperties` to encapsulate the magic strings.  Please add any new ones that may be necessary.

Reply:
I did not know about that class, thanks for pointing it out.

it's a good cleanup, added to the scope of the PR beyond just the original line move: 4af088c